docs/how-to/troubleshoot_user_registration_issues.md

```diff
@@ -0,0 +1,98 @@
+# Troubleshoot subscriber registration issues
+This guide provides step-by-step troubleshooting actions to remediate subscriber registration issues issues. We hope you don't need this guide. If you encounter an issue and aren't able to address it via this guide, please raise an issue [here][Bug Report].
+
+## 1. Subscriber registration failed with `REGISTRATION-REJECT-EVENT` error
+
+### Symptoms
+The `juju debug-log sdcore-gnbsim-k8s/leader` command reports an error in the UE registration:
+```console
+handling event: REGISTRATION-REJECT-EVENT	{"component": "GNBSIM", "category": "SimUe", "supi": "imsi-001010100007487"}
+```
+
+### Recommended Actions
+This error is typically raised when the UE is not provisioned in SD-Core or the link between AMF and AUSF or between AUSF and UDM/UDR is not established.
```

Comment:
This sentence is not a "Recommended Action" but an explanation, if those are needed, please add a section for it.